### ISSUE TITLE
Reminder about open Notes

### ISSUE THREAD

**Issue by Fizzie41** [openstreetmap/openstreetmap-website#3477]
### Description

Wondering if there could be any way of modifying OSM Notes so that, if the Note hasn't been resolved after a set period e.g. 3 months, the system sends you a reminder that it is still open?

Naturally, this wouldn't work if the Note has been added by a non-mapper e.g. by somebody using OSMAND, but it would help with those Notes that have been created as an aide-memoir by active mappers, who have then forgotten about them! :-)

We're currently working through the Australian Notes & finding quite a few from very experienced mappers that were set-up a number of years ago to "check on this" but they're still sitting there, & in some cases, the person has openly admitted that yes, they forgot about it.

Thanks,

**Comment by tomhughes** [comment]
Well the whole point of a note normally is for somebody that doesn't know the area to ask somebody that does to investigate so I'm not sure how reminding the reporter helps?

**Comment by Fizzie41** [comment]
For those experienced mappers who have added a note to remind themselves to do "this" but have then forgotten about it.

**Comment by mmd-osm** [comment]
Why don’t you use existing services such as https://my-notes.osm-hr.org/

**Comment by Fizzie41** [comment]
Because they're not my notes!

**Comment by tordans** [comment]
> Well the whole point of a note normally is for somebody that doesn't know the area to ask somebody that does to investigate  

I remember an analysis about OSM notes that found that many notes are closed by the same person that opened them. From this it was deduced that one key use case for notes are personal notes that the mapper herself then works on. Unfortunately I did not find the blogpost anymore.

>That’s what’s been recommended on the wiki page: https://wiki.openstreetmap.org/wiki/Notes#Finding_your_notes
Source code is available. Tweak it as needed.

That looks like a valid workaround. It would not send notifications, but one could easily highlight notes that are "older than 30 days" in the list.

**Comment by Fizzie41** [comment]
Carrying on from highlighting aging notes, it would probably make sense to then automatically delete them altogether when they reach "12 months" old. Seriously, if they haven't been actioned by then, what is the likelihood of it still happening in the future?

**Comment by tomhughes** [comment]
Why not leave them? Are they harming you in some way?

**Comment by Fizzie41** [comment]
No, but people keep complaining that there's too much cluttering up the database, so surely these must contribute to that clutter?

**Comment by SomeoneElseOSM** [comment]
>  Seriously, if they haven't been actioned by then, what is the likelihood of it still happening in the future?

In my experience, quite high.  I regularly get update from notes I added months or years ago, including one just this morning.

**Comment by danieldegroot2** [comment]
> I remember an analysis about OSM notes that found that many notes are closed by the same person that opened them.

See [joost schouppe/diary/43057](https://www.openstreetmap.org/user/joost%20schouppe/diary/43057) (found it mentioned in https://github.com/openstreetmap/iD/issues/5172 )
